web/src/components/shared/Ads/Ad.tsx

```diff
@@ -55,15 +62,42 @@ const ErrorBlock: FC<{ text: string }> = ({ text }) => (
   </Box>
 );
 
+const WarningBlockMobile: FC<{ unactiveReason: string }> = ({ unactiveReason }) => (
```

Comment:
```suggestion
interface WarningBlockMobileProps {
  unactiveReason: string
}

const WarningBlockMobile: FC<WarningBlockMobileProps> = ({ unactiveReason }) => (
```

Reply:
Можно `children` использовать. Это хорошая практика

```suggestion
const WarningBlockMobile: FC = ({ children }) => (
```

Reply:
done